Sources/XcodeProj/Objects/Files/PBXGroup.swift

```diff
@@ -126,6 +128,28 @@ public extension PBXGroup {
             .first(where: { $0.name == name })
     }
 
+    /// Returns the file in the group with the given path, based on locations on-disk.
+    /// - Note: Paths function as-in Xcode; with respect to the group's `PBXSourceTree`.
+    /// Internally, identity is calculated based on normalized absolute paths for all files involved; i.e. actual
+    /// location in the file system.
+    ///
+    /// - Parameters:
+    ///   - path: a file path to search for.
+    ///   - sourceRoot: path to project's source root.
+    /// - Returns: an existing reference to that file, or `nil` if not found
+    func file(with path: Path, sourceRoot: Path) -> PBXFileReference? {
+        func absolutePath(for fileRef: PBXFileElement) -> Path? {
+            try? fileRef.fullPath(sourceRoot: sourceRoot)
```

Comment:
Going with the convention of the other methods, then, and "just pass in `$(SRCROOT)` here and everything should work". Tests seem to show it does